addOns/commonlib/commonlib.gradle.kts

```diff
@@ -0,0 +1,128 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+plugins {
+    `maven-publish`
+    signing
+}
+
+group = "org.zaproxy.addon"
+
+version = "0.1.0"
+description = "A common library, for use by other add-ons."
+
+zapAddOn {
+    addOnName.set("Common Library")
+    addOnStatus.set(AddOnStatus.ALPHA)
```

Comment:
Does it really matter for this add-on? Should be changed to release?

Reply:
+1 for release